draft-ietf-rtcweb-jsep.xml

```diff
@@ -3640,6 +3640,20 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         </list>
       </t>
 
+      <t>For each RTCP packet received, the following steps MUST be
```

Comment:
We probably need a more general treatment of how to route RTCP. That is, how do we route normal RTCP packets in a bundle scenario?